emcc.py

```diff
@@ -1010,7 +1010,8 @@ def add_link_flag(i, f):
         file_suffix = get_file_suffix(arg)
         if file_suffix in SOURCE_ENDINGS + OBJECT_FILE_ENDINGS + DYNAMICLIB_ENDINGS + ASSEMBLY_ENDINGS + HEADER_ENDINGS or shared.Building.is_ar(arg): # we already removed -o <target>, so all these should be inputs
           newargs[i] = ''
-          if file_suffix.endswith(SOURCE_ENDINGS):
+          #if file_suffix.endswith(SOURCE_ENDINGS):
```

Comment:
these unnecessary comments need to be removed for `flake8` to pass.

Reply:
Should be fixed.